docker/docker-compose.ci.yml

```diff
@@ -48,11 +47,20 @@ services:
     image: unlock
     user: node
     working_dir: /home/unlock/paywall
+    expose:
+      - 3001
+    environment:
+      CI: "${CI}"
+      HTTP_PROVIDER: "${HTTP_PROVIDER}"
+    command: ["npm", "run", "start"]
   integration-tests:
     image: unlock-integration
     user: node
     environment:
       UNLOCK_HOST: "unlock"
+      PAYWALL_URL: "${PAYWALL_URL}"
```

Comment:
I am not sure this is a good approach. Since we know that the service will be at `paywall` (that is how docker works?) could we just use these values rather than force ourselves to set them?

Reply:
For consistency, you would probably need to use `PAYWALL_HOST: "paywall"` and actually build the URL from the host?